### PR TITLE
Basic Space Page to Showcase

### PR DESCRIPTION
**Pull request by joshuanianji**
## Description

This PR adds some some basic things to the Space Showcase, aiming to mimic the original Antd website with the components we have.

I didn't include any tests, nor have done many in the past, but if you want to show me how I can definitely try it out.
This page will slightly change after making the default spacing direction horizontal. I think I should submit a PR for that before merging this one, though.

## Screenshots

Screen Shot 2020-10-13 at 8 45 26 PM
Screen Shot 2020-10-13 at 8 45 29 PM

## Action Items

- [x] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [x] This PR is as small as it can be, and only implements on thing (not many)
- [x] I have manually tested my feature
- [ ] I have written a "good" amount of tests (including visual and unit tests)

### CODE REVIEW

**Comment by supermacro** [comment]
LGTM 👍 ... just need to fix formatting (`./scripts/format-elm-files.sh`) and this is good to merge! 

Note that you might want to rebase on top of master as I just pushed some commits yesterday and today to release the [theming api](https://github.com/supermacro/elm-antd/releases/tag/4.2.0).

**Comment by joshuanianji** [comment]
Alright, the code is formatted and I synced my fork with the upstream branch! 

I'll wait until the #55 is merged, then I'll do one final change to the spacing examples, which is just deleting a couple of `Space.direction Space.Horizontal`.

**Comment by joshuanianji** [comment]
I made the changes to the spacing examples. Let me know if it's good to merge!

**Review by supermacro — APPROVED** [review]

**Pull request merged**
